com.unity.multiplayer.mlapi/Runtime/Core/NetworkedBehaviour.cs

```diff
@@ -38,6 +38,8 @@ internal enum NExec
             Client = 2
         }
 
+        private const string StandardRpc_ChannelName = "STDRPC";
```

Comment:
we might want to call out that we have plans around channel names and this hardcoded channel name string might not be mandatory.

Reply:
I agree with this, and will this to a new RPCQueue Revisions Jira Task.